### PR TITLE
Make html basis

### PR DESCRIPTION
**Pull request by item4**
I try to make lxml.html module basis. But I am not good for internal of lxml. I will make draft PR. Please review that?

### CODE REVIEW

**Comment by item4** [comment]
p.s. I write this code based on https://lxml.de/apidoc/_modules/lxml/html.html

**Comment by item4** [comment]
any review?

File: lxml-stubs/html/__init__.pyi [hunk 1]
```diff
@@ -1,3 +1,104 @@
...
+
+if TYPE_CHECKING:
+    from .. import _AnyStr, _AnySmartStr, _BaseParser, _Element
+    from .. import HTMLParser as _HTMLParser
+    from .. import XMLParser as _XMLParser
+
+
+def __fix_docstring(s: str) -> str: ...
+
+
```
Comment (scoder) [inline lxml-stubs/html/__init__.pyi:16]: This is an implementation detail. Let's not expose it here.
```suggestion
```

**Review by scoder — COMMENTED** [review]
Looks good to me. Thanks, and sorry for the long delay.

**Pull request merged**